### ISSUE TITLE
achilles final merge fails on individual analysis failure

### ISSUE THREAD

**Issue by fdefalco** [OHDSI/Achilles#656]
Issues such as the one found at the following link occur frequently within the community of users.

https://forums.ohdsi.org/t/running-achilles-issue/16579/2

As part of execution the failure of an analysis should be caught and the final merge updated to prevent total failure on the failure of individual analyses.

**Comment by faisalsafar99** [comment]
I executed the below command on R:
> remotes::install_github("OHDSI/Achilles")

then I got message error as below: 
Error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Resolving timed out after 10000 milliseconds

I think the issues from fire wall and proxy! 
Is there any command that I can go over the fire wall and proxy?

**Comment by acumenus** [comment]
> Issues such as the one found at the following link occur frequently within the community of users.
> 
> https://forums.ohdsi.org/t/running-achilles-issue/16579/2
> 
> As part of execution the failure of an analysis should be caught and the final merge updated to prevent total failure on the failure of individual analyses.

![Screenshot 2023-03-13 at 9 34 17 PM](https://user-images.githubusercontent.com/126921002/224871800-636ba365-538c-4db9-a257-21110d828625.png)

Is this what the failure looks like?  I'm trying to find the right place to post my issue.

**Comment by MaximMoinat** [comment]
@acumenus Indeed, it seems like analysis 716 failed, which in turn made the merge fail.

@fdefalco Any progress on this. Related to what I posted under #697

**Comment by acumenus** [comment]
@alondhe - has this happened to you?  Do you know of any fix other than what was posted on the Forums last year?:
![Screenshot 2023-03-14 at 12 43 52 PM](https://user-images.githubusercontent.com/126921002/225077789-cbc547a0-3900-44c0-99e6-fd99a33be973.png)
